packages/shared/components/popups/ManageVotingPowerPopup.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="typescript">
+    import { TextType } from 'shared/components/enums'
+    import { Button, Text, TextHint, AssetAmountSliderInput } from 'shared/components'
+    import { selectedAccount } from '@core/account'
+    import { handleError } from '@core/error/handlers/handleError'
+    import { setVotingPower } from '@core/governance/actions'
+    import { localize } from '@core/i18n'
+    import { checkActiveProfileAuth } from '@core/profile'
+    import { visibleSelectedAccountAssets } from '@core/wallet'
+    import { closePopup } from '@auxiliary/popup'
+
+    const asset = $visibleSelectedAccountAssets?.baseCoin
+
+    let assetAmountSliderInput: AssetAmountSliderInput
+    let rawAmount = $selectedAccount?.votingPower
+
+    function handleBack(): void {
+        closePopup()
+    }
+
+    async function handleConfirm(): Promise<void> {
+        try {
+            await assetAmountSliderInput?.validate()
+            await checkActiveProfileAuth(async () => {
+                await setVotingPower(rawAmount)
+                closePopup()
+            })
+        } catch (err) {
+            handleError(err)
+            closePopup()
+        }
+    }
+</script>
+
+<Text type={TextType.h4} classes="mb-3">{localize('popups.manageVotingPower.title')}</Text>
+<Text type={TextType.p} secondary classes="mb-5">{localize('popups.manageVotingPower.body')}</Text>
+<div class="space-y-4 mb-6">
+    <AssetAmountSliderInput bind:this={assetAmountSliderInput} bind:rawAmount {asset} />
```

Comment:
This doesn't seem to work properly. I get the following error when trying to use it:
```
Error: [big.js] Invalid number
    at parse (big.js:138:1)
    at new Big (big.js:109:1)
    at Big (big.js:92:1)
    at convertToRawAmountFromMetadata (convertToRawAmount.ts:24:1)
    at convertToRawAmount (convertToRawAmount.ts:12:1)
    at $$self.$$.update (AssetAmountSliderInput.svelte:32:1)
    at update (index.mjs:1089:1)
    at flush (index.mjs:1060:1)
```

Reply:
This was only present in german versions, but Mark fixed it in the meantime